libxcrypt.yaml

```diff
@@ -0,0 +1,64 @@
+package:
+  name: libxcrypt
+  version: 4.4.36
+  epoch: 0
+  description: "Modern library for one-way hashing of passwords"
+  copyright:
+    - license: GPL-2.0-or-later AND LGPL-2.1-or-later
+
+environment:
+  contents:
+    packages:
+      - wolfi-base
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - autoconf
+      - automake
+      - pkgconf
+      - libtool
+      - gettext
+
+pipeline:
+  # Using Fetch instead of git checkout
+  # @kanini : When we retire libcrypt from glibc, it will need to get built earlier than git, as git (indirectly) depends on libcrypt.
```

Comment:
```suggestion
  # @kaniini : When we retire libcrypt from glibc, it will need to get built earlier than git, as git (indirectly) depends on libcrypt.
```